app/components/app_session_summary_component.rb

```diff
@@ -6,7 +6,11 @@ class AppSessionSummaryComponent < ViewComponent::Base
       <%= @session.location.name %>
     </h3>
 
-     <%= govuk_summary_list(rows:, classes:) %>
+    <% if @session.school? %>
```

Comment:
```suggestion
    <% if session.school? %>
```

Reply:
Yeah because of the `attr_reader`, if anything we should also update the line above to `session.location.name`.

Reply:
Fixed! And updated `session.location.name` too.